README.md

```diff
@@ -42,6 +39,39 @@ use_frameworks!
 
 Then run `pod install` with CocoaPods 0.36 or newer.
 
+## Adding Runes as a Framework Dependency ##
+
+Runes can also be used as a low-overhead dependency for defining these
+operators for your own types. After linking to `Runes.swift`, you will be able
+to define implementations for your types, and your users will be able to just
+import your framework to use these operators.
+
+### [Carthage] ###
+
+```
+github "thoughtbot/runes"
+```
+
+Then run `carthage update`. Once you've checked out the required version, you
+can link `Carthage/Checkouts/Runes/Source/Runes.swift` directly into your
+framework.
+
+### [CocoaPods] ###
+
+First, you will need to make sure that Runes is added as a submodule for your
+repo. Then, you'll need to make sure to enable submodule support for your
+source directive:
+
+```ruby
+s.source = { :git => "https://github.com/thoughtbot/Argo", :tag => "v#{s.version}", :submodules => true }
```

Comment:
if not then maybe say "for example" in more words so people know that you showing them how to implement this in their own frameworks